### ISSUE TITLE
Issue with HomePod

### ISSUE THREAD

**Issue by SimonSeku** [milgra/macmediakeyforwarder#69]
When iTunes is connected with HomePod, this app will change the skip button to do nothing, and the pause button to skip. Back button is the same.

**Comment by milgra** [comment]
I have to get a HomePod first to investigate this problem :(

**Comment by SimonSeku** [comment]
Makes sense, if you need someone with a HomePod to test anything, let me know!

**Comment by milgra** [comment]
Sure, thank you!
